src/io/MmpldWriter.cpp

```diff
@@ -105,7 +105,8 @@ void MmpldWriter::readXML(XMLfileUnits& xmlconfig)
 	numSites = query.card();
 	global_log->info() << "[MMPLD Writer] Number of sites: " << numSites << endl;
 	if(numSites < 1) {
-		global_log->warning() << "[MMPLD Writer] No site parameters specified." << endl;
+		global_log->fatal() << "[MMPLD Writer] No site parameters specified." << endl;
+		Simulation::exit(48973);
```

Comment:
I'm still not happy with these "exit codes". -> #195

Reply:
for grepping them, it is actually really handy :)
so definitely postponed ;)